src/components/common/ControlsBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles(() => ({
+  controlsContainer: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  control: {
+    margin: 10,
+  },
+}));
+interface ControlsBarProps {
+  children: React.ReactNode;
+}
+const ControlsBar: React.FC<ControlsBarProps> = ({ children }: ControlsBarProps) => {
```

Comment:
@0qy this class could possibly be replaced with MUI Toolbar (). NB the MUI examples aren't great but [this article looks helpful ](https://stackoverflow.com/questions/52653103/what-is-appbar-vs-toolbar)

Reply:
Just tried out the Toolbar works the same so I will go ahead and delete that component.